includes/hooks/template.php

```diff
@@ -56,7 +56,7 @@
 add_action( 'sensei_archive_before_course_loop', array( 'Sensei_Course', 'archive_header' ), 10, 0 );
 
 // add the course image above the content
-add_action( 'sensei_course_content_inside_before', array( $sensei->course, 'course_image' ), 10, 1 );
+add_action( 'sensei_course_content_inside_before', array( $sensei->course, 'course_image' ), 30, 1 );
```

Comment:
This was necessary because otherwise the other functions attached to this hook meant that the output order would be course meta -> course image -> pricing instead of course meta -> pricing -> course image, and you ended up with something like this:
![Screen Shot 2019-07-17 at 7 02 48 PM](https://user-images.githubusercontent.com/1190420/61417584-8848b200-a8c5-11e9-8bc5-2cfc14e2b16b.jpg)